src/tower/game_view.py

```diff
@@ -31,6 +32,12 @@ def setup(self):
         self.scene = arcade.Scene.from_tilemap(self.tile_map)
 
         self.towers = arcade.SpriteList()
+        
+        self.towers.append(sprites.Tower("tower1", "A basic tower", 1, [sprites.AttackSpec("Basic Attack", "A basic attack", 1, 1, 1)]))
```

Comment:
This is a placeholder so it's fine for now but for future Tower instantiations I think it'd be more readable to do it on multiple lines, especially the AttackSpec

Reply:
lint will change it anyways, line is way too long :D